src/main/java/org/broadinstitute/hellbender/tools/exome/CnvCaller.java

```diff
@@ -27,6 +28,8 @@
  *
  * Note:  Assumes that the incoming target file is in log_2(CR)
  * Note:  Assumes that the incoming segment means are CR.
+ * Note: Assumes that the input @{link ReadCountCollection} contains only one sample -- calls will be made for only one
```

Comment:
@samuelklee @LeeTL1220 come to think of it, it might make sense to have an abstract parent class for tools that do something for each sample/column of a read column collection, so that child classes would just have to implement `processColumn(final int column)` or something like that.